src/models/GivenEventModel.ts

```diff
@@ -0,0 +1,58 @@
+import type {
+  InferAttributes,
+  InferCreationAttributes,
+  Sequelize,
+} from 'sequelize';
+import { DataTypes, Model } from 'sequelize';
+import type { AccountId, Address, BigIntString } from '../core/types';
+import getSchema from '../utils/getSchema';
+import { COMMON_EVENT_INIT_ATTRIBUTES } from '../core/constants';
+import type { IEventModel } from '../events/types';
+
+export default class GivenEventModel
+  extends Model<
+    InferAttributes<GivenEventModel>,
+    InferCreationAttributes<GivenEventModel>
+  >
+  implements IEventModel
+{
+  public declare accountId: AccountId; // The `tokenId` from `Transfer` event.
+  public declare receiver: AccountId; // The `tokenId` from `Transfer` event.
```

Comment:
I think the comments do not apply here, right?

Reply:
true, will remove